src/chrome-finder.ts

```diff
@@ -33,13 +33,13 @@ export function darwin() {
 
   execSync(
       `${LSREGISTER} -dump` +
-      ' | grep -i \'google chrome\\( canary\\)\\?.app$\'' +
+      ' | grep -i \'google chrome\\( canary\\)\\?.app .*$\'' +
```

Comment:
now this matches nothing on pre-catalina versions like mine :)

how about let's just remove the `$` business completely (leave as `?.app'`) and do the rest of our filtering in JS where it's a little easier to reason about?

Reply:
You are absolutely right, I didn't check that one on the Mojave mac mini before I pushed it up. I reverted that change, and put a check to pushing to the installations array to make sure items are unique. Between that and the canAccess() function that will ensure that only valid, unique chrome apps are found and added to the array.